### DISCUSSION TITLE
Copyright dates

### DISCUSSION THREAD

**Discussion by davecramer** [pgjdbc/r2dbc-postgresql#368] [General]
I just noticed that we put start-end dates in the copyright messages. My preference is to only put the start date in as this will not require changing files every year.
Also I noticed that in the lates PR [#353 ] that the new files have 2017-2020. My preference would be to use the date the file was first introduced.

Thoughts?

**Comment by mp911de** [comment]
In the Spring eco-system, we're used to using the inception year and the year a file as touched for the last time. I don't know where the rule comes from. Looking at some Apache projects (Spark, Log4j2), they don't have the `Copyright 2017-2020 the original author or authors.` line.

* RxJava uses `2016-present`
* guava uses the start year.

I don't have a strong opinion since I wasn't able to find anything that would give guidance from an Apache License side.

**Comment by davecramer** [comment]
My recommendation is just the start year. I just want to avoid the whole changing the copyright date because the year changed.

**Comment by mp911de** [comment]
We can do that. I filed #369.
